docs/Gallery.md

```diff
@@ -35,3 +35,10 @@ This page highlights some visualizations of SpECTRE simulations.
 Here's an image of an adaptively refined mesh:
 
 <img src="https://drive.google.com/thumbnail?id=1ivR2Wbu_RYvHih098waKi6jqx637pSUn&sz=w600"/>
+
+## Binary Black Holes
+
+Visualization of apparent horizons of an equal mass non-spinning BBH. The
+colorful surface represents the lapse and the arrows depict the shift vector.
```

Comment:
Could you expand the description a bit? I think it would be nice to have this page give an overview of what spectre can do. So it would be nice to include a brief description of the BBH problem, and some features that make spectre unique for this problem. Maybe just expand this a little bit in this PR and then we merge this, and we can iterate on the descriptions more later?

I also suggest you include the name of the person who made the visualization, e.g. "Visualization by: A. Carpenter (CSU Fullerton)". We should also include references to papers if there are any.

Reply:
Ah ok, I was just copying what was done for the AMR image with the short description and no credits. Should I change that one as well?

Reply:
Sure feel free to expand that section as well if you want. Larry or I will edit it later too, so it doesn't have to be perfect.

Reply:
I decided to only do the BBH one. You/Larry can edit the AMR one since you've been the ones working on it.